hippylib/modeling/PDEProblem.py

```diff
@@ -15,10 +15,12 @@
 
```

Comment:
Consider splitting this file in 3 files.
One for the base class, one for `PDEVariationalProblem` and one for `TimeDependentPDEVariationalProblem`.

Update copyright accordingly

Reply:
I'm happy to do so, but this seems to be inconsistent with how other portions of the library are implemented. For example, in `misfit.py`, the base `Misfit` class is present as well as child classes such as `DiscreteStateObservation` and (now) `MisfitTD`.

Reply:
The reason I'm suggesting splitting is that the implementation of `PDEProblem` derived classes tends to be much longer than the implementation of specific derived classes of `Misfit`.

This is a matter of taste, but in general I don't like files with too much code in it.

Reply:
@uvilla Sorry for the delay... Understandable -- I've migrated the derived classes to their own files, so I think this should be resolved now?